armada-ths/ais
# Banquet flow

Created a banquet dummy flow.
Changed on_deletion behavior for BanquetTables to stop them from deleting banquetattendants.

Sidenote:
I'm not sure how @ellwes commits ended up here, I need to check with her if thats ok

To test:
* ! run tests locally ! this is important, because Travis broke!
* host server
* goto localhost:8000/fairs/2017/banquet/
* make sure you have at least 1 banquet attendant
* press the "sit attendants" button on the top
* make sure the attendants got sitted at tables. (Will use existing tables with correct sizes, or create new ones of size 8 as needed)
